### PR TITLE
Package asai.0.1.1

### PR DESCRIPTION
**Pull request by favonia**
### `asai.0.1.1`
A library for constructing and printing compiler diagnostics
This package offers an implementation of compiler diagnostics and supports multiple backends for displaying diagnostics to the end user. Currently, the package comes with three backends: GitHub Actions workflow commands, UNIX terminals, and LSP (Language Server Protocol).

---
* Homepage: https://github.com/RedPRL/asai
* Source repo: git+https://github.com/RedPRL/asai.git
* Bug tracker: https://github.com/RedPRL/asai/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/asai/asai.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+description:
+  "This package offers an implementation of compiler diagnostics and supports multiple backends for displaying diagnostics to the end user. Currently, the package comes with three backends: GitHub Actions workflow commands, UNIX terminals, and LSP (Language Server Protocol)."
+maintainer: ["favonia <favonia@gmail.com>"]
+authors: ["The RedPRL Development Team"]
+license: "Apache-2.0"
+homepage: "https://github.com/RedPRL/asai"
+bug-reports: "https://github.com/RedPRL/asai/issues"
+depends: [
+  "dune" {>= "3.1"}
+  "ocaml"
```
Comment (haochenx) [inline packages/asai/asai.0.1.1/opam:13]:
```suggestion
  "ocaml" {>= "5.0"}
```

**Review by haochenx — COMMENTED** [review]
Everything looks good to me except the fact that it's recommended to add lower bounds for OCaml.

**Comment by favonia** [comment]
Thanks. Updated the upstream https://github.com/RedPRL/asai/commit/c207da3441ce280dc82e3b3c2f6dfbf45242389b (but this PR still points to the old source archive if that's okay)

**Comment by haochenx** [comment]
Thanks!

> but this PR still points to the old source archive if that's okay

I think it's usually okay for us if it's okay for you. It's definitely always best to have matching opam files in opam-repository and the tarball, but there are many cases this isn't the case. It is strongly recommended to upstream the change for future releases (as you just did) though.

This PR looks all good to me now.

**Review by haochenx — APPROVED** [review]

**Comment by favonia** [comment]
> Thanks! You may want to announce this on https://discuss.ocaml.org, where we have a Community category and an `announce` tag for this purpose.

Yes! This is one of the other two packages I was talking about. 😃

**Pull request merged**